query-service-impl/src/test/java/org/hypertrace/core/query/service/pinot/ExecutionContextTest.java

```diff
@@ -138,6 +140,8 @@ public void testFiltersWithLiterals() {
 
   @Test
   public void testReferencedColumns() {
+    long currentTimeInMillis =
+        Instant.ofEpochSecond(TimeUnit.SECONDS.convert(1, TimeUnit.HOURS)).toEpochMilli();
```

Comment:
- This isn't the current time, variable name is incorrect
- Your converting an hour into seconds into an instance to convert back to millis. `MILLISECONDS.convert(1, HOURS)` would give you the same result.

Reply:
Unresolving. See second bullet (and our naming is still not matching how we're using it - it was called start time, but you use it for end time).